glass/fields.py

```diff
@@ -329,16 +329,18 @@ def getcl(cls, i, j, lmax=None):
 
     Parameters
     ----------
-    cls : list of array_like
+    cls:
         List of angular power spectra in *GLASS* ordering.
-    i, j : int
-        Combination of indices to return.
-    lmax : int, optional
+    i:
+        Indices to return.
+    j:
+        Indices to return.
+    lmax:
         Truncate the returned spectrum at this mode number.
 
     Returns
     -------
-    cl : array_like
+    cl:
```

Comment:
The return values for napoleon + numpy requires specifying both the name and type of the variable - https://numpydoc.readthedocs.io/en/latest/format.html#returns. Kind of annoying, but right now sphinx does not render the return values properly - 

<img width="760" alt="image" src="https://github.com/user-attachments/assets/19044ffc-5a2d-4af9-958c-8c42c9fec6c7">

Reply:
I think the issue here is that it treats "Returns" and "Return type" as independent information. In which case we probably want to remove "Returns" from the docstrings?

Reply:
So yeah it seems that we are forced to specify types in the docstrings. I think I'm going to move the returns bits to the summary sections.

Reply:
> Kind of annoying, but right now sphinx does not render the return values properly -

`cl` isn't a type, it think it is

Reply:
Have moved `Returns` and that has works alright, I think